client/test/task-list.test.js

```diff
@@ -14,7 +14,9 @@ describe('Task List', function() {
 
   it('should show a table of the pollers of the task list', async function () {
     var [taskListEl] = await taskListTest(this.test),
-        lastAccessBase = moment().startOf('hour')
+      lastAccessBase = moment().startOf('hour')
+
+    await Promise.delay(1);
```

Comment:
any reason to have delay in test?

Reply:
Test fails without the delay. Ideally if I had more time to fix this properly I would convert these integration tests to unit tests as it is much more reliable and stable.